src/utils/AuthService.ts

```diff
@@ -60,15 +63,204 @@ function reloadTimerOnExpiresIn(
     userManager: UserManager,
     expiresIn: number
 ) {
-    // TODO: Can we stop doing it in the hash for implicit flow ? To make it common for both flows
-    // Not allowed by TS because expires_in is supposed to be readonly
-    // @ts-ignore
-    user.expires_in = expiresIn;
-    userManager.storeUser(user).then(() => {
+    const updatedUser: User = {
```

Comment:
I tested a bit it feels like it breaks the automatic token renewal with custom values... This part of the code is quite touchy it would be nicer it you don't change code and just disable eslint rules here